Formula/rubberband.rb

```diff
@@ -3,7 +3,7 @@ class Rubberband < Formula
   homepage "https://breakfastquay.com/rubberband/"
   url "https://breakfastquay.com/files/releases/rubberband-1.9.0.tar.bz2"
   sha256 "4f5b9509364ea876b4052fc390c079a3ad4ab63a2683aad09662fb905c2dc026"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```
The top of the source and header files all say `or (at your option) any later version`.

See https://github.com/breakfastquay/rubberband/search?q=license